pkg/resource/git.go

```diff
@@ -247,7 +247,7 @@ func (r *GitRepo) Refs() []string {
 	refs := []string{}
 
 	if err != nil {
-		logger.Error("could not list refs: error=%q", err)
+		r.Error(logger.ScanError, "could not list refs: error=%q", err)
```

Comment:
Hmm it feels weird having ScanError inside the resources like this. Just because it wouldn't be a scan error if refs failed in some other context.

Reply:
We can add ErrorCodes, thats not a big issue.